SR2020/2019-08-31-announce-SR2020.md

```diff
@@ -0,0 +1,29 @@
+---
+to: Previously registered interested team leaders
+subject: SR2020 Registration and Kickstart Announcement
+---
+
+Hi,
+
+We're pleased to announce that registration for Student Robotics 2020 is now
+open. If you would like to compete, please complete this [application form][signup-form]
+with the details of your team. All future emails about SR2020 will be sent there.
+
+This year we're hosting three Kickstart events, all on **TODO DATE**:
+
+ * at Thread's offices in London
+ * at the University of Southampton
+ * in Cambridge
+
+We're expecting to run with a similar number of teams to SR2019, so we're sure
+that places will go fast. We expect to confirm places at the end of September,
```

Comment:
I'd suggest rewording this final sentence a bit. Personally i'd rather it be that we confirm places as we go, but we have a fixed deadline to apply. Thoughts?

Reply:
I'm not sure what the benefit of explicitly confirming as we go is? My assumptions here are:
- we'd confirm as soon as we could anyway, we're limited by confirmation of venue size and complete kits to hand out
- a hard deadline is meaningless as we'd happily accept applications after that if we had space

My aim here is to provide a feeling for when teams should expect to hear back by, with the previous sentence being the motivation for getting on with it.

Reply:
Theoretically, but I believe teams generally prefer to have as much notice as possible, as it helps with arranging getting to the kickstart events.

Reply:
In my opinion the current wording gives us the most options. There is no reason we couldn't confirm teams earlier than the end of September but if for some reason we can't confirm as we go we have given teams a deadline that is achievable for us.